nss-ice-phantoms/WorkforceManagement
# Rr employees

# Description

Builds and populates Employee edit form.  Does NOT yet post edits.


## Type of change

Please delete options that are not relevant.



# Testing Instructions for Change Made

1.  git fetch --all
2.  git checkout rr-employees
3.  Run BangazonWorkforce
4.  Verify Edit link by any employee in the list takes you to the edit page and displays First Name, Last Name, Full Name, Department Name, a list of Computers (the employee's currently assigned computer should be marked with ***) and two lists of Training Programs, those the employee is enrolled in and those they are not enrolled in.


# Checklist:
